tools/depcheck/src/allowlist.rs

```diff
@@ -142,13 +142,18 @@ pub const EXTRA_DATAGEN_DEPS: &[&str] = &[
     "databake-derive",
     "elsa",
     "erased-serde",
+    "fraction",
     "icu_codepointtrie_builder",
-    "itertools",
     "itoa",
+    "itertools",
     "matrixmultiply",
     "ndarray",
+    "num",
```

Comment:
num is the meta crate for all num-* crates, we shouldn't need to depend on it?

Reply:
done

Reply:
no?

Reply:
it was done, but after this it is added because `tidy` is failing. Fraction is depending on `num`

```
[[package]]
name = "fraction"
version = "0.14.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
checksum = "59a78dd758a47a7305478e0e054f9fde4e983b9f9eccda162bf7ca03b79e9d40"
dependencies = [
 "num",
]```

Reply:
ah that's unfortunate